apps/airdrop/src/App/AnotherWays.tsx

```diff
@@ -24,6 +25,12 @@ export const AnotherWays: React.FC<AnotherWaysProps> = (props) => {
   const metamask = (window as MetamaskWindow)?.ethereum;
   const columns = !metamask && !keplr ? "1fr 1fr" : "1fr 1fr 1fr";
 
+  useEffect(() => {
+    if (typeof props.reset === "boolean") {
```

Comment:
Just to check I'm understanding this: The `reset` prop is not really a boolean; it's like a signal which every time it gets changed will tell the component to uncheck the TOS checkbox?
That's pretty cool! I had to read it a few times to get my head around it though :smile: